src/RuleSet/AtRuleSet.php

```diff
@@ -45,6 +45,8 @@ public function atRuleName()
 
     /**
      * @return string
+     *
+     * @deprecated since V8.8.0, will be removed in version 9.0.0. Use the specific getter instead.
```

Comment:
This doesn't have a specific getter for the `$arguments` property, so we should add one first.